docs/man/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 set(core
-	gendiff.1 rpm2cpio.8
+	gendiff.1 rpm2cpio.8 rpm2archive.8
```

Comment:
Eh, rpm2archive.8 is already in the `extra` list below.

Reply:
I wonder if we should move it to `core` or keep it there. Probably doesn't matter at all since both lists are joined together anyway :smile:

Reply:
Oh... except `core` is actually used to look for enabled plugin material while `extra` isn't.

Reply:
OK, we want this to be in `core` since `extra` is used for optional features. Therefore, just drop it from `extras` and we're good to go.

Reply:
Oh, I simply missed the one in extra. Thanks for spotting.